hazelcast/src/main/java/com/hazelcast/client/impl/proxy/ClientMultiMapProxy.java

```diff
@@ -328,6 +328,12 @@ public UUID addLocalEntryListener(@Nonnull EntryListener<K, V> listener) {
         throw new UnsupportedOperationException("Locality for client is ambiguous");
     }
 
+    @Nonnull
+    @Override
+    public UUID addLocalEntryListener(@Nonnull EntryListener<K, V> listener, boolean includeValue) {
+        throw new UnsupportedOperationException("Locality for client is ambiguous");
```

Comment:
Can you add a dummy test to cover this code line?

Reply:
Sure, pushed test.